layers/+lang/emacs-lisp/packages.el

```diff
@@ -46,6 +47,19 @@
 (defun emacs-lisp/post-init-eldoc ()
   (add-hook 'emacs-lisp-mode-hook 'eldoc-mode))
 
+(defun emacs-lisp/init-auto-compile ()
+  (use-package auto-compile
+    :defer t
+    :init
+    (progn
+      (setq auto-compile-display-buffer nil
+            auto-compile-use-mode-line t
+            auto-compile-mode-line-counter t)
```

Comment:
I think it's better in `:init`. If a deferred package uses `:config` it becomes more difficult for a user to override the values of those variables. Certainly anything that requires something to be defined needs to go in `:config`, but I think setting custom variables should be done in `:init`, unless you know something I'm missing